### DISCUSSION TITLE
Create Xcode project

### DISCUSSION THREAD

**Discussion by aremmell** [aremmell/libsir#485] [Ideas]
If we've got VS Code and Visual Studio project files, we should have Xcode as well.

**Comment by johnsonjh** [comment]
Did we ever figure this out?  I looked and it's non-trivial (at least to me).

**Comment by aremmell** [comment]
I got about half-way there. You can check out the `xcode` branch. The UI is so bad that a person who's been doing almost nothing besides using computers for 20 years can't figure it out...
